src/calendar/actions/calendar-event/get-all-events-for-current-person.ts

```diff
@@ -0,0 +1,30 @@
+import { Fetcher } from '../../../common/api';
+import { endpoints } from '../../api/calendar-event/endpoints';
+import { ICalendarEvent } from '../../entities';
+import { ISearchResult } from '../../../common';
+import { IGetCalendarEventForCurrentPersonInputDto } from '../../types';
+import { errorFactory, CalendarErr } from '../../errors/errors';
+
+export const getCalendarEventForCurrentPerson = async (
```

Comment:
```suggestion
export const getCalendarEventsForCurrentPerson = async (
```

On peut mettre la méthode au pluriel étant donné que cette méthode retourne une liste.